src/main.cpp

```diff
@@ -15,11 +16,105 @@ template <typename T> std::chrono::milliseconds inMilliseconds(T duration)
   return std::chrono::duration_cast<std::chrono::milliseconds>(duration);
 }
 
+void init_cart_info(CartogramInfo *cart_info,
```

Comment:
This function contains code that was originally in the main function, so I think leaving it here is appropriate.

Reply:
Correct. Since every class allows for a constructor and we are running the `init_cart_info` function every time we are creating a new object of `CartogramInfo` class, I feel the function materials are best suited to be placed inside class constructor. Also, the `main.cpp` is very very messy with lots of things. I would personally prefer it to be less crowded.